theme/layout/theme.liquid

```diff
@@ -23,12 +23,29 @@
   {% render 'nav' %}
 
   <main role="main">
+    {% if shop.metafields.accentuate.show_twitch_stream %}
+      <div
```

Comment:
I'm cool with this, but would prefer we throw it in a snippet to keep this file small.

Reply:
Fixed, thank you!